lib/score-board.js

```diff
@@ -1,31 +1,37 @@
-var _ = require('lodash');
-
 function ScoreBoard(list){
   this.list = list;
   this.scores = [];
 }
 
 ScoreBoard.prototype.loadStoredScores = function(){
-  if(this.scores.length > 0){
-    this.scores = localStorage.getItem('scores');
-    var storedScores = this.scores.split(",");
-    var list = this.list;
-    var descendingScores = _.sortBy(storedScores).reverse();
-    descendingScores.forEach( function(score){
-      var scoreItem = document.createElement('li');
-      scoreItem.innerText = score;
-      list.appendChild(scoreItem);
+  var storedScores = uniq(localStorage.getItem('scores').split(","));
+  this.scores = storedScores;
+  var list = this.list;
+  var scores = this.scores;
+  var descendingScores = storedScores.sort(function(a, b){return b-a;});
+  list.innerHTML = "";
+  descendingScores = descendingScores.slice(0,5);
+  descendingScores.forEach( function(score){
+    var scoreItem = document.createElement('ol');
+    scoreItem.innerText = score;
+    list.appendChild(scoreItem);
+    scores.push(score);
   });
-  }
 };
 
+function uniq(a) {
+    return a.sort().filter(function(item, pos, ary) {
+        return !pos || item !== ary[pos - 1];
```

Comment:
I know this function is pretty self-explanatory but more verbose variable names might make it even easier to understand... what do you think?